applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -108,6 +120,7 @@ def GetDefaultParameters(cls):
             "domain_size" : -1,
             "echo_level": 0,
             "analysis_type": "linear",
+            "time_integration": "implicit",
```

Comment:
[in StructuralMechanics this is called `time_integration_method`](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/python_scripts/python_solvers_wrapper_structural.py#L12)

```suggestion
            "time_integration_method": "implicit",
```

Reply:
Ok.